src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/Frame.xaml

```diff
@@ -5,15 +5,205 @@
+                        </Trigger>
+                        <Trigger Property="Header" Value="{x:Null}">
+                            <Setter TargetName="Icon" Property="Margin" Value="0" />
+                            <Setter TargetName="HeaderPresenter" Property="Visibility" Value="Collapsed" />
+                        </Trigger>
+                        <Trigger Property="JournalEntryUnifiedViewConverter.JournalEntryPosition" Value="Current">
+                            <Setter TargetName="Icon" Property="Content">
+                                <Setter.Value>
+                                    <TextBlock Text="{StaticResource FrameMenuiItemCurrentGlyph}" FontFamily="{DynamicResource SymbolThemeFontFamily}" />
+                                </Setter.Value>
+                            </Setter>
+                        </Trigger>
+                        <MultiTrigger>
+                            <MultiTrigger.Conditions>
+                                <Condition Property="MenuItem.IsHighlighted" Value="True"/>
+                                <Condition Property="JournalEntryUnifiedViewConverter.JournalEntryPosition" Value="Forward"/>
+                            </MultiTrigger.Conditions>
+                            <Setter TargetName="Icon" Property="Content">
+                                <Setter.Value>
+                                    <TextBlock Text="{StaticResource FrameMenuItemArrowRightGlyph}" FontFamily="{DynamicResource SymbolThemeFontFamily}" />
+                                </Setter.Value>
+                            </Setter>
+                        </MultiTrigger>
+                        <MultiTrigger>
+                            <MultiTrigger.Conditions>
+                                <Condition Property="MenuItem.IsHighlighted" Value="True"/>
+                                <Condition Property="JournalEntryUnifiedViewConverter.JournalEntryPosition" Value="Back"/>
+                            </MultiTrigger.Conditions>
+                            <Setter TargetName="Icon" Property="Content">
+                                <Setter.Value>
+                                    <TextBlock Text="{StaticResource FrameMenuItemArrowLeftGlyph}" FontFamily="{DynamicResource SymbolThemeFontFamily}" />
+                                </Setter.Value>
+                            </Setter>
+                        </MultiTrigger>
+                        <Trigger Property="IsEnabled" Value="False">
+                            <Setter TargetName="Icon" Property="TextElement.Foreground" Value="{StaticResource FrameMenuItemForegroundDisabled}"/>
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <ControlTemplate x:Key="DefaultFrameTemplateKey" TargetType="{x:Type Frame}">
```

Comment:
This is not actually default, so we should rename this. We can rename this to `FrameTemplateKey`.